### ISSUE TITLE
Document id(true) in README

### ISSUE THREAD

**Issue by SaphiLC** [bryanjhv/slim-session#24]
Docs are missing the ID function, i think its important to know you can regenerate the session with that

**Comment by bryanjhv** [comment]
That variant is for emergency. Regenerating ID keeps data on server, but user may have a bad connection and loose all the data, giving more work to the server. Tell me a practical use case for that.

**Comment by SaphiLC** [comment]
practical use? to prevent session fixation attack,
```
bool session_regenerate_id ([ bool $delete_old_session = false ] )
```
its recommended to run regenerate_id after any important interaction with the server (login in, changing password...) and at random times

**Comment by bryanjhv** [comment]
Please link to the source about that, because the docs about [session_regenerate_id](https://secure.php.net/session_regenerate_id) don't say it.  
For session fixation please encrypt your sessions instead of monkey-patching an existing one.

**Comment by SaphiLC** [comment]
in case you dont know what session fixation is (looks like you dont) https://www.owasp.org/index.php/Session_fixation

from https://www.booleanworld.com/8-critical-security-issues-avoid-website/
```
Issue #3: Problems in authentication and session management
(...)
Not regenerating sessions: When an user is logged in or logged out, you should regenerate the session identifier associated with the user, in order to prevent a “session fixation” attack. Suppose, an attacker logs into a website from a public computer, notes the session identifier, and then logs out. When another user logs in from the same computer, the same identifier is being reused. Thus, the attacker can send requests from another computer with the same identifier to determine if another user has logged in. This allows for a complete compromise of the user’s account.
```
from https://github.com/expressjs/session/issues/425 (even php haters agree)
```
I quite like how the did it in PHP (I know, the horror): http://php.net/manual/en/function.session-regenerate-id.php

default is to keep the old session data
```
i could continue adding info but you can just search "how to prevent session fixation in php"

anyways i already changed to another session package, i cant trust this one anymore, hope you the best (no hate)

**Comment by bryanjhv** [comment]
Yeah, [illuminate/session](https://packagist.org/packages/illuminate/session) is one of the best existing ones, and manages much things this package does not, but this one was meant to be simple and usable, only with native sessions (as described in the README).
Security things are being addressed in #21 so looks like you either didn't read other issues before posting yours.
